.github/workflows/build_idf_examples.yml

```diff
@@ -0,0 +1,28 @@
+name: Build ESP-IDF USB examples
+
+on:
+  schedule:
+    - cron: '0 0 * * SAT' # Saturday midnight
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        idf_ver: ["release-v5.0", "release-v5.1", "release-v5.2", "latest"]
```

Comment:
I haven't been able to Google a solution for this. Is there a way to use wildcards in YAML matrix lists. Would be nice if the lists could expand itself using a `release-v5.*` wildcard.

Currently, the tinyusb `idf_component.yml` is supposed to be supported on all IDF v5.x versions. If we could use a wildcard, then we won't need to keep updating this file on every IDF 5.x release.

Reply:
Yep, have created the jira for that.